CMakeLists.txt

```diff
@@ -41,4 +41,4 @@ if(BuildTests)
     add_subdirectory(tests)
 endif()
 
-# add_subdirectory(example)
+add_subdirectory(example)
```

Comment:
Can you hide this behind an option? just this breaks projects that use cmake fetchcontent_declare to import the client.

Something like:
`FileWatch_Build_Example`?

Thank you